atomic_reactor/plugins/build_sources.py

```diff
@@ -0,0 +1,89 @@
+"""
+Copyright (c) 2019 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+from __future__ import print_function, unicode_literals, absolute_import
+
+import tempfile
+
+from atomic_reactor.build import BuildResult
+from atomic_reactor.constants import PLUGIN_CONTAINER_SOURCE_BUILD_KEY
+from atomic_reactor.plugin import BuildStepPlugin
+from atomic_reactor.plugins.pre_reactor_config import get_value
+
+
+class SourceContainerBuilderPlugin(BuildStepPlugin):
+    """
+    Build source container image using
+    https://github.com/containers/BuildSourceImage
+
+    Image https://quay.io/repository/ctrs/bsi should be pushed to image stream
+    on OCP instance and should be used in `source_image_builder`
```

Comment:
Let's make sure this is consistent with the plugin name.

Reply:
Reworded